science/grig/Portfile

```diff
@@ -15,8 +15,10 @@ master_sites            sourceforge:groundstation
 
 depends_build           port:pkgconfig
 depends_lib             port:gtk2 \
-                        port:hamlib
+                        path:lib/libhamlib.dylib:hamlib
 
-checksums           rmd160  01deadaea1ae2e6ed3e27ada07ad4557039ddf8c \
-                    sha256  be8687418fb23efa0468674c3fdd15340fed06eef09be9de21106cc17e033c25 \
-                    size    621728
+checksums               rmd160  01deadaea1ae2e6ed3e27ada07ad4557039ddf8c \
+                        sha256  be8687418fb23efa0468674c3fdd15340fed06eef09be9de21106cc17e033c25 \
+                        size    621728
+
+patchfiles              patch-hamlib4.diff
```

Comment:
I didn't update the revision because this patch affect only people that use hamlib-devel and therefore if `grig` is already installed the re-build will be triggered by `rev-upgrade`. if it is a new installation no problem.

Reply:
rev-upgrade is a backstop. We try not to rely on it. Regardless, you're changing the binary, that means we bump the revision.

Reply:
I know, `rev-upgrade` is the last resort but to avoid useless rebuild like in this case I avoided it. Anyway, I bump the revision.